### ISSUE TITLE
multiline SyntaxError

### ISSUE THREAD

**Issue by laerus** [xonsh/xonsh#2036]
this appears to happen only with prompt_toolkit
```
$ print(
. 
$ <--- hitting return here
  File "<string>", line None
SyntaxError: <xonsh-code>:0:0: EOF in multi-line statement
$ 
```

**Comment by scopatz** [comment]
Thanks for reporting @laerus.  I am not sure what is going on here, but I haven't done much work on the PTK side of things...

**Comment by gforsyth** [comment]
hey @laerus, can you see if #2045 fixes this for you?

**Comment by gforsyth** [comment]
Fixed in #2045.
